doc/user_guide/ensembleModel.tex

```diff
@@ -0,0 +1,229 @@
+\section{Ensemble Models}
+\label{sec:ensembleModel}
+
+% TODO
+% Introduction
+% Data Objects are the Key
+% Model Definition
+
+In most RAVEN MultiRun steps, a sampler provides a variety of inputs to a model, whose outputs then produce a
+set of responses that can be used in many ways.  However, sometimes a single model is insufficient to produce
+the response we want.  When this happens, the EnsembleModel opens up a new vista of simulation options.
+
+The EnsembleModel is a powerful tool for chaining multiple models together, with outputs of some models
+becoming inputs for others.  Whether this is adding a preprocessing or postprocessing model, or linking
+multiple physics-based models together, the EnsembleModel provides the functionality to see many models
+together as a single model in a MultiRun step.
+
+Note that the Ensemble Model combines multiple models for \emph{each sample}; that is, the whole ensemble of
+models is run for each sample taken by RAVEN.  If instead you want to postprocess a batch of sampled data, try
+the PostProcessor models instead.
+
+\subsection{Introduction: The Ensemble Model}
+The key to the Ensemble Model is careful definition of the inputs and outputs of each model.  When an
+Ensemble Model is defined in a RAVEN run, RAVEN will automatically scan the inputs and outputs of each model
+and determine the right order to evaluate the models in (or \emph{graph}).
+
+\subsection{Example: ballistics and impact}
+% TODO add an image showing these two models, their inputs and outputs
+As a basic example, consider two physics models.  The first is a ballistics code, used to determine the
+kinetic energy $E$ of a ball when it hits the ground, with a path depending on its initial height $y_0$, initial velocity
+$v_0$, mass $m$ and initial angle $\theta_0$.  We'll assume we're near the Earth's surface and aside from the initial
+launch height, we're launching over a flat surface.  This can be represented as a functional $E(y_0,v_0,m,\theta_0)$.
+
+The second physics model is an impact code that estimates the diameter of the crater $D$ made when a ball
+impacts.  With a few simplifying assumptions ($D$ is under 1 km, the impact is near Earth's surface, the
+impact is in dry sand with fixed density), size $D$ can be determined as a function of the ball's kinetic
+energy when impacting $E$, its mass $m$, and its radius $r$: $D(E,m,r)$.
+
+We can see that the input $E$ for the impact model is calculated by the ballistics code.  In RAVEN the
+ensemble model will automatically detect this, and run the ballistics code before the impact code in each
+iteration.
+
+In general, the EnsembleModel also has some finite capacity for resolving circular dependencies using Picard
+iteration.  See the manual for more information on this capability.
+
+Setting up an ensemble model can be a tricky process, so we'll walk through the input using our example
```

Comment:
I would remove the expression "tricky process". I would change this with:
```The EnsembleModel input construction is a bit more complex than the one for the other Model entities, so we'll walk through the input using our example```

Reply:
Changed.